modules/oxide/src/less/skins/ui/tinymce-5/skin.less

```diff
@@ -77,6 +77,10 @@
     margin-bottom: -1px;
   }
 
+  &.tox-tinymce-inline .tox-editor-container {
+    overflow: hidden;
```

Comment:
Won't this cause clipping issues if the header has a shadow via the `@tinymce-box-shadow` variable for example?

Reply:
Fair, if it was like that in 5.x then lets leave it like that.